sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/tests/BlobTests.cs

```diff
@@ -18,6 +18,7 @@ namespace Microsoft.Azure.WebJobs.Extensions.Storage.Blobs
     public class BlobTests
     {
         private const string TriggerQueueName = "input-blobtests";
+        private const string ConnectionName = "AzureWebJobsStorage";
```

Comment:
If you're able to follow my connection resolution suggestions above, we won't need this constant.

Reply:
Updated